src/SFA.DAS.EmployerAccounts.Web/Startup.cs

```diff
@@ -1,27 +1,31 @@
-﻿using Microsoft.Azure;
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens;
+using System.Linq;
+using System.Security.Claims;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using System.Web.Mvc;
+using Microsoft.Azure;
+using Microsoft.IdentityModel.Protocols;
 using Microsoft.Owin;
 using Microsoft.Owin.Security;
 using Microsoft.Owin.Security.Cookies;
+using Microsoft.Owin.Security.Notifications;
+using Microsoft.Owin.Security.WsFederation;
+using Newtonsoft.Json;
 using NLog;
 using Owin;
 using SFA.DAS.Authentication;
 using SFA.DAS.EmployerAccounts.Configuration;
+using SFA.DAS.EmployerAccounts.Interfaces;
+using SFA.DAS.EmployerAccounts.Models.Account;
 using SFA.DAS.EmployerAccounts.Web;
 using SFA.DAS.EmployerAccounts.Web.Authentication;
+using SFA.DAS.EmployerAccounts.Web.Models;
```

Comment:
This whole configuration needs tidying up and extracting out of here.  I assume this will be done after initial testing.

Reply:
We need to add this TechDebt Task

Reply:
Will ask Chris to create a tech debt task for this